purchases/src/main/kotlin/com/revenuecat/purchases/google/BillingWrapper.kt

```diff
@@ -898,6 +899,19 @@ internal class BillingWrapper(
         )
     }
 
+    private fun trackFeatureNotSupportedIfNeeded() {
```

Comment:
Actually should we rename this to `trackProductDetailsNotSupported` instead? And also rename the counter name, since currently only tracks the product details feature.

Alternatively, we could keep this name and have the feature as a parameter here and a tag in the counter, but not sure if we want to track other features...

Reply:
renamed 👍